mantidproject/mantid
# Remove RangeSelectors when PreviewPlot is in x squared

Fixes [#11365](http://trac.mantidproject.org/mantid/ticket/11365).

To test:
- Open any indirect interface with a plot that has range selectors
- Switch X axis to Squared
- Notice RangeSelectors are no longer shown
